pycaret/internal/pycaret_experiment/time_series_experiment.py

```diff
@@ -1,8 +1,14 @@
+from pandas.core.frame import DataFrame
```

Comment:
remove?

Reply:
Yes, we can probably remove it. I will check and remove.

Reply:
removed